UnityProject/Assets/Scripts/Input System/KeybindManager.cs

```diff
@@ -515,6 +518,31 @@ public void LoadKeybinds()
 				ResetKeybinds();
 				ModalPanelManager.Instance.Inform("Unable to read saved keybinds.\nThey were either corrupt or outdated, so they have been reset.");
 			}
+
+			
+			// Check if new hotkeys has been added to the default dict
```

Comment:
It's a good start but what if something gets added and something else gets removed?

If you use LINQ (stick `using System.Linq;` at the top) you can do something like this to find any dictionary keys that are different:
```
var keysDictionary1HasThat2DoesNot = dictionary1.Keys.Except(dictionary2.Keys);
var keysDictionary2HasThat1DoesNot = dictionary2.Keys.Except(dictionary1.Keys);
```
Then if there are any differences you can add/remove entries as necessary.

Reply:
Nice, I will use that as soon as my local repo unfucks itself.

Reply:
what type does this Except method returns? Is it just another dictionary?

Reply:
Done. Works on my end, but tell me if that's what you asked.